### ISSUE TITLE
Small adjustment to link "How to Meet WCAG 2.1"

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag#412]
_From @jake-abma on June 8, 2018 15:40_

At: https://www.w3.org/TR/WCAG21/#wcag-2-1-supporting-documents there's a link refering to a "How to Meet WCAG 2.1" document.

This is actually a link to the "How to Meet WCAG 2" document WITHOUT the ".1"

In the filter you could choose 2.1 as wel as 2.0 or 2.1 addition only, but the document itself serves both.

The link therefore should be [How to Meet WCAG 2](https://www.w3.org/WAI/WCAG21/quickref/)

_Copied from original issue: w3c/wcag21#958_

**Comment by alastc** [comment]
This is now a link to the quickref, which now defaults to 2.1, so I think it is accurate now. There are filters when you get there.

Closing, @jake-abma please re-open if I've misunderstood.
